docroot/profiles/custom/cgov_site/themes/custom/cgov/cgov_common/templates/content/home_landing/field.html.twig

```diff
@@ -0,0 +1,3 @@
+{% for item in items %}
```

Comment:
Ah HA! so this is how you cleaned up the other field templates. This should most certainly definitely NOT be hidden in the home_landing if we are changing all field templates.

Are we changing all field templates? This should be a conversation for all and not a little side effect -- basically I don't want any regressions. If we do this it should be at the root of the templates. (I am not saying no, I am just saying not here... I certainly have overridden enough of these to warrant it...)

Reply:
Would this template propagate to other content types? I thought, since it's in the homelanding folder, that it would be contained to homelanding content types, but that may not be correct.

This worked as a good field template reset, and then I added field specific alterations, rather than writing templates for every field. After I hit 11 `field--paragraph` templates I thought to give it a try and it worked well.

Reply:
Yes, by making this it sets the template for all fields everywhere for this theme. Our folders are just for our organization. 

I am not saying it is a bad thing, just a bad place...

Reply:
Well, I think the field should be reset to prevent future headaches. I'd like to hear reasons why it shouldn't be reset as well as where it should live. Not that I have an opinion. I'm still figuring out the project structure.

Reply:
We agreed it should be overwritten, but moved out (and it was with another PR)